tests/test_core.py

```diff
@@ -7295,8 +7295,14 @@ def test_noexitruntime(self):
   def test_minmax(self):
     self.do_runf(path_from_root('tests', 'test_minmax.c'), 'NAN != NAN\nSuccess!')
 
-  def test_locale(self):
-    self.do_run_in_out_file_test('tests', 'test_locale.c')
+  def test_localeconv(self):
+    self.do_run_in_out_file_test('tests', 'core', 'test_localeconv.c')
+
+  def test_newlocale(self):
+    self.do_run_in_out_file_test('tests', 'core', 'test_newlocale.c')
+
+  def test_setlocale(self):
+    self.do_run_in_out_file_test('tests', 'core', 'test_setlocale.c')
 
```

Comment:
these feel excessive to me in `core` as I don't think opt levels matter for them? how about `other`?

Reply:
Sure, but I think there are a whole lot of tests that fall into that category no?    For example this existing `test_locale` was in the wrong place according your assessment?    How should we make this decision about which test suite a give test belongs in?  (I'm sure I filed a bug about this discussion at some point but I can't seem to find it now).

Reply:
My reasoning is that `core` runs on various opt levels, so tests that want that should be there. Otherwise, it can be in `other`.

A codegen test for exceptions would be in `core`, for example, to see the optimizer can handle it. A libc issue like this could be in `other`.